scripts/js/onboard/index.js

```diff
@@ -0,0 +1,69 @@
+const { ApiPromise, WsProvider, Keyring } = require('@polkadot/api');
+const fs = require('fs')
+
+const run = async () => {
+  try {
+    const wsProvider = new WsProvider('ws://0.0.0.0:9944');
```

Comment:
I would like this to be passed as well, since we might extend this script to onboard or upgrade programatically against the development environment

Reply:
I think this is fine but it's also very easy to just pass this from `init.sh`

Reply:
Yeah I am already passing a few values, why not.